configuration/configuring-language/en.md

```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
 +++
 # Configuring Language
 
-You can change your language by going to the Budgie Menu, opening Settings, then clicking Region and Language.
+It is easy to change the language that was selected during the installation of Solus.
```

Comment:
Doesn't necessarily have to be during the installation of Solus though. Fairly common to do post-install adding of languages for bi-lingual usage.